src/braket/circuits/angled_gate.py

```diff
@@ -231,6 +234,9 @@ def __repr__(self):
             f"'qubit_count': {self.qubit_count})"
         )
 
+    def __hash__(self):
+        return hash((self.name, self.angle_1, self.angle_2, self.qubit_count))
+
 
 class TripleAngledGate(Gate, Parameterizable):
```

Comment:
Why didn't we implement hash for this?

Reply:
This got added recently. Missed that it was pulled in from main.